app/utils/Process.js

```diff
@@ -160,13 +174,14 @@ class Process extends EventEmitter {
     }
 
     _compressImage = algorithm => {
+
```

Comment:
Create the newPath and copy the file here, because here is where the process begin.

Reply:
The key problem is that on every algorithm, it will create a new copy, so, for example, the second algorithm created a smaller output size, than the third, it will overwrite the second algorithm output. It need to run once, so thats why I made this. I can create a flag if already made a copy, for that. Is it a good aproach?